db/migrate/20210930192755_change_anonymous_default_from_donations.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeAnonymousDefaultFromDonations < ActiveRecord::Migration
+  def change
+    change_column_default :donations, :anonymous, false
+    change_column_null :donations, :anonymous, false
```

Comment:
Do you think it would be better to change this in a separate PR after we merge the first part?

Reply:
I do! Merge the first part and update prod run the script to correct all the old ones and merge this and update prod again. Honestly, I'm not sure we even need this line as the first line should fix it.